doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+source code file. In this example, we invoke the Python interpreter to run a
+python script. Create the run settings using the ``create_run_settings()`` function
+provided by the ``exp`` (``Experiment``) object. Specify the path to the source code file (script) to the argument
+``exe_args`` and the run command to the ``exe`` argument.
+
+.. code-block:: python
+
+  model_settings = exp.create_run_settings(exe="python", exe_args="model_multidb_example.py")
+
+Next, set the number of nodes and tasks per node for the ``Model``. For the application (``Model``),
+the number of nodes determines the distribution of computational tasks while tasks per node
+specifies how many individual instances of the ``Model`` (tasks) can run on each node. In this
+example, we specify to SmartSim that we intend to execute the ``Model`` once on a single node.
+
+.. code-block:: python
+
+  model_settings.set_nodes(1)
+  model_settings.set_tasks_per_node(1)
+
+Next, create an instance of the ``Model`` object using the ``create_model()`` function of the
+``exp`` object. Pass in the ``model_settings`` object, which was configured earlier, as an argument
+to the ``create_model()`` function and assign to the variable ``model``.
+
+.. code-block:: python
+
+  model = exp.create_model("tutorial-model", model_settings)
+  exp.generate(model_example, overwrite=True)
+
+Next, launch the model instance using the exp.start() function.
+
+.. code-block:: python
+
+  exp.start(model, block=True, summary=True)
+  exp.generate(model, overwrite=True)
+
+
+Finally, use the ``Experiment.stop()`` function to stop the database instances and print the
+workflow summary with ``exp.summary()``.
+
+.. code-block:: python
+
+  exp.stop(colocated_orch, clustered_orch)
+  logger.info(exp.summary())
+
+We have finished writing the experiment workflow. Continue reading to write the application workflow.
+
+Integrating SmartRedis Multi-Database Functionality into the Model
+------------------------------------------------------------------
+
+Next in the example, add SmartRedis multi-database functionality to the ``Model``.
+Navigate to the file specified in the ``exe_args`` argument of the ``create_run_settings()``
+function, ``exe_args="/path/to/model_multidb_example.py"``.
+
+To begin, learn to retrieve data from the launched databases by using a SmartRedis Client
+and the database identifier. In the script, we do not have access to the
+``exp.get_address()`` function. To establish a connection with the launched databases,
+we still need to initialize a new SmartRedis client for each ``Orchestrator`` with the address of
+the launched database.
+
+Previously, we added the ``db_identifier`` argument when creating the two ``Orchestrators``.
+When a database is created with a ``db_identifier`` name, SmartSim creates an environment
+variable named ``SSDB`` for each database with the ``db_identifier`` variable value suffixed to
```

Comment:
Again, I think we should remove references to SSDB here. It's a bit too much information for the user and not needed at this stage

Reply:
I disagree. It's good that the user knows what's going on even if they aren't dealing with it themselves. Perhaps move it to a note to suggest it's less critical?